Client/src/api/api.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { LOCAL_SERVER, API_ADDRESS, QUERY_TYPE } from './constants';
+
+function getLocalData() {
+  const [apiData, setApiData] = useState('');
+  const [currentLocation, setcurrentLocation] = useState();
+  const [token, setToken] = useState();
+  const [weatherdata, setweatherData] = useState();
+
+  const AUTH_TOKEN = token;
+  const AUTH = {
+    Authorization: `Bearer ${AUTH_TOKEN}`
+  };
+
+  // Calls server and gets a token
+
+  useEffect(() => {
+    axios.get(LOCAL_SERVER).then(result => setToken(result.data));
```

Comment:
We will probably talk about it later but I suggest to keep all api related calls in a dedicated service so that none of your React components would know about axios or any other http providers